### ISSUE TITLE
Desktop Restaurant Navbar Contrast Issue

### ISSUE THREAD

**Issue by erennovakim** [icssc/ZotMeal#725]
Fix the Navbar on desktop to not be transparent and have black, transparent gradient behind navbar. See above image for correct [Figma](https://www.figma.com/design/Jci9nJ1AkwnEllPF0CF7Sn/PeterPlate-Desktop-and-Mobile-Design?node-id=0-1&p=f&t=3Id7JF9ucoOyTduu-0) and below for what is on the website. This presents an accessibility issue and does not pass WCAG3 contrast and visibility standards.

<img width="1440" height="270" alt="Image" src="https://github.com/user-attachments/assets/cd02288d-38b4-474f-91f0-2c13cd9b85c4" />

<img width="1271" height="187" alt="Image" src="https://github.com/user-attachments/assets/7a682c75-9ff2-4e84-8f26-76b9af3d4a12" />

**Comment by PraneshJha1038** [comment]
@erennovakim  could you specify the file responsible for the home page

**Comment by EightBitByte** [comment]
Though not on main yet, this has been fixed on `dev`.
